src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/DataFrameReaderTests.cs

```diff
@@ -75,6 +76,11 @@ public void TestSignaturesV2_3_X()
             string textFile = $"{TestEnvironment.ResourceDirectory}people.txt";
             Assert.IsType<DataFrame>(dfr.Text(textFile));
             Assert.IsType<DataFrame>(dfr.Text(textFile, textFile));
+
+            string randomName = $"table_{new Random().Next(100000, 999999).ToString()}";
```

Comment:
why does this need to be a random table name ?

Reply:
If the files already exist then spark will fail, there used to be a workaround but it was removed with spark 3.0 spark.sql.legacy.allowCreatingManagedTableUsingNonemptyLocation and if the option is set in spark 3.0 it throws an exception.

Reply:
`DataFrameReader.Table(..)` looks like an alias for `SparkSession.Table(..)`.  Can we do the same thing we do for that test?

```csharp
            _spark.Range(10).CreateOrReplaceTempView("testView");
            Assert.IsType<DataFrame>(_spark.Table("testView"));
```

Reply:
I've copied it now